### DISCUSSION TITLE
Uncontrolling growth of elements in lists of asks and bids in Depth Cache Manager

### DISCUSSION THREAD

**Discussion by chubatrik** [oliver-zehentleitner/unicorn-binance-local-depth-cache#44] [Q&A]
### Description
When I create a DC and taking out bids and asks, an amount of elements in theese lists are growing without any border. The amount always starts about 1000 and growing up. 

### Results:
When I start the program: 
![Screenshot_1](https://github.com/user-attachments/assets/a055bef3-3a89-48f6-b80e-9aa21c3a3c54)
After 15 minutes of working:
![image](https://github.com/user-attachments/assets/04ef0133-c408-4a31-9a18-4c1acae3d0d7)

### Source code:
```coin = "ethusdt"
ubldc = BinanceLocalDepthCacheManager(exchange="binance.com-futures", depth_cache_update_interval=100)
ubldc.create_depthcache(coin)

while True:
    try:
        try:
            asks = ubldc.get_asks(coin)
            bids = ubldc.get_bids(coin)
        except DepthCacheOutOfSync:
            asks = "Out of sync!"
            bids = "Out of sync!"
        print(f"Bids: {len(bids)}, asks: {len(asks)}")
        time.sleep(1)
    except (Exception, KeyboardInterrupt) as k:
        print(k)
        ubldc.stop_manager()
        print(f"finished")
        exit()

**Comment by oliver-zehentleitner** [comment]
Continue here: https://github.com/LUCIT-Systems-and-Development/unicorn-binance-local-depth-cache/issues/45
